ui/app/templates/components/transit-key-action/rewrap.hbs

```diff
@@ -9,7 +9,16 @@
     <div class="field">
       <label for="ciphertext" class="is-label">Ciphertext</label>
       <div class="control is-expanded">
-        {{textarea name="ciphertext" class="textarea" id="ciphertext" value=ciphertext}}
+        {{ivy-codemirror
+            value=ciphertext
+            valueUpdated=(action (mut ciphertext))
```

Comment:
this seems to work fine without the id in both the UI and the tests, but out of curiosity, do you know why we had an id before? i presume it was for tests or css?

Reply:
I think this was for accessibility - it mapped the label to the textarea and clicking on the label would focus the textarea.

Reply:
It was for tests and the "copy" function (which is now just referencing the data value directly instead of the DOM)